LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2018 UCSC Computational Genomics Lab
+   Copyright 2019 Global Alliance for Genomic Health
```

Comment:
s/b "Global Alliance for Genomics and Health"

Reply:
@dglazer Good catch. Updated PR